cantabular-import/dp-cantabular-filter-flex-api.yml

```diff
@@ -15,9 +15,15 @@ services:
         ports:
             - 27100:27100
         restart: unless-stopped
+        depends_on:
+            - kafka-1
+            - kafka-2
+            - kafka-3
+            - mongodb
         environment:
             BIND_ADDR:                   ":27100"
-            SERVICE_AUTH_TOKEN:          $SERVICE_AUTH_TOKEN
+            SERVICE_AUTH_TOKEN:           $SERVICE_AUTH_TOKEN
+            KAFKA_ADDR:                  "kafka-1:19092,kafka-2:19092,kafka-3:19092"
```

Comment:
Should this line be:
            KAFKA_ADDR:                         "kafka-1:9092"

, please ask David Subiros

Reply:
I think you're getting confused with the component test yml files. The answer is 'no' to the above questions... check the other yml files in this directory you'll see they're set up the same.